lib/facilities/ppms/v1/response.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
```

Comment:
👍  Definitely like the refactoring to move the pagination into the response class

Reply:
ty